### DISCUSSION TITLE
Update the usage of Discussion in replacement for the RFC Issue

### DISCUSSION THREAD

**Discussion by sleepylee** [nimblehq/android-templates#318] [RFCs]
### Why
Since we decided to move to Discussion, it is yet to be updated on our WIKI nor README. A process update always requires a document update; in this case, we need to guide the participant on how to use Discussion or Issues effectively.

### Expectation
https://github.com/nimblehq/android-templates/wiki
https://github.com/nimblehq/android-templates/blob/develop/README.md

these 2 must be updated

**Comment by luongvo** [comment]
IMO, this process is quite simple so we can add it to README only + reference to the contribution guidelines from Compass https://nimblehq.co/compass/intro/contribute 🤔

**Comment by toby-thanathip** [comment]
IMO, I prefer keeping it in WIKI only

We explain our voting process in WIKI already, so seems to be a good place for other processes too

**Comment by toby-thanathip** [comment]
@ryan-conway Since we already integrated #267 & #378, would you agree that this RFC can be resolved? 💭

**Reply by ryan-conway** [reply]
Yes, this discussion should be resolved by these issues 🙏🏻
